### ISSUE TITLE
CRASH WITH SHADERS BY LOOKING AT A DISK DRIVE

### ISSUE THREAD

**Issue by silverace71** [refinedmods/refinedstorage#2989]
#### Issue description:
GAME CRASH WHILE RUNNING SHADERS
#### What happens:
look at disk drive while running sildurs vibrant shaders lite will close the game and give you a crash report
#### What you expected to happen:
nothing
#### Steps to reproduce:
1.get mods
2.get shaders
3.put a 64k disk into a disk drive
4.look at it and crash
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft:1.16.5
- Forge:36.1.18
- Refined Storage:1.9.12

Does this issue occur on a server? [yes/no]
yes
#### If a (crash)log is relevant for this issue, link it here:
https://pastebin.com/Gtdfhx8S

**Comment by hexandcube** [comment]
I have a similar issue, the **game doesn't crash**, but **the disk drive isn't being rendered** at all, and I keep getting an **error in the chat**, every time I look at the disk drive: 
![image](https://user-images.githubusercontent.com/41151156/120717323-a45c9900-c4c7-11eb-8d8f-8cf8f1419e18.png)

```
[CHAT] CCL Caught an exception rendering a block. See the log for info.
```

**Minecraft Log (No crash reports have been generated):** https://pastebin.com/02MgZhJ9

- **Minecraft** 1.16.5
- **Forge** 36.1.23
- **Refined Storage** 1.9.12

**Mod Pack**: FTB Presents Direwolf 20 1.16.5 1.10.1 (Server)

No shaders, no OptiFine.

**Recording of the issue:**
https://user-images.githubusercontent.com/41151156/120717744-5ac07e00-c4c8-11eb-98c9-ece16cb9e3dd.mp4

**Comment by necauqua** [comment]
This is fixed by #3003

**Comment by raoulvdberge** [comment]
Duplicate of #2963
